applications/DeepSpeed-Chat/training/step3_rlhf_finetuning/training_scripts/single_node/sweep/run_lora_sweep.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Let's add a very small `README.md` to these sweep folders explaining what they are used for. I think most users won't use this but we want this of course when do testing/evaluation of new features, etc.

Reply:
Added READMEs for step 1 and 3 sweeping scripts and create a section for characterization sweeps in the main training README.